pkg/api/validation/dynakube/oneagent.go

```diff
@@ -23,6 +23,8 @@ Use a nodeSelector to avoid this conflict. Conflicting DynaKubes: %s`
 
 	errorVolumeStorageReadOnlyModeConflict = `The DynaKube specification specifies a read-only host file system while OneAgent has volume storage enabled.`
 
+	errorPublicImageWithWrongConfig = `The DynaKube specification specifies a custom (and therefor probably a public) image in combination with a mode that needs write permissions for volume mounts.`
```

Comment:
```suggestion
	errorPublicImageWithWrongConfig = `The DynaKube specification specifies a custom (most likely public) image in combination with a mode that needs write permissions for volume mounts.`
```

Reply:
I think the information in the warning is confusing and incorrect: https://github.com/Dynatrace/dynatrace-operator/pull/4346#discussion_r1932070866

Reply:
Well I agree to be honest it may be a little too technical